tests/test_samples.py

```diff
@@ -1339,3 +1339,126 @@ def test_old_gui():
+                        chains.skew().loc[chain, :])
+        assert_allclose(chains.get_group(chain).mad()
+                        .drop(('chain', '$n_\\mathrm{chain}$')),
+                        chains.mad().loc[chain, :])
+        assert_allclose(chains.get_group(chain).sem()
+                        .drop(('chain', '$n_\\mathrm{chain}$')),
+                        chains.sem().loc[chain, :])
+        assert_allclose(chains.get_group(chain).corr()
+                        .drop(('chain', '$n_\\mathrm{chain}$'))
+                        .drop(('chain', '$n_\\mathrm{chain}$'), axis=1),
+                        chains.corr().loc[chain, :])
+        assert_allclose(chains.get_group(chain).cov()
+                        .drop(('chain', '$n_\\mathrm{chain}$'))
+                        .drop(('chain', '$n_\\mathrm{chain}$'), axis=1),
+                        chains.cov().loc[chain, :])
+        assert_allclose(chains.get_group(chain).corrwith(mcmc)
+                        .drop(('chain', '$n_\\mathrm{chain}$')),
+                        chains.corrwith(mcmc).loc[chain, :]
+                        .drop(('chain', '$n_\\mathrm{chain}$')))
+        q = np.random.rand()
+        assert_allclose(chains.get_group(chain).quantile(q)
+                        .drop(('chain', '$n_\\mathrm{chain}$')),
+                        chains.quantile(q).loc[chain, :])
+
+    assert_allclose(mcmc.mean().drop(('chain', '$n_\\mathrm{chain}$')),
+                    chains.mean().mean())
+
+    for col in mcmc.columns:
+        if 'chain' not in col:
+            for chain in [1, 2]:
+                i = mcmc.chain == chain
+                assert_allclose(mcmc.loc[i, col].mean(),
+                                chains[[col]].mean().loc[chain, :])
+                assert_allclose(mcmc.loc[i, col].std(),
+                                chains[[col]].std().loc[chain, :])
+                assert_allclose(mcmc.loc[i, col].median(),
+                                chains[[col]].median().loc[chain, :])
+                assert_allclose(mcmc.loc[i, col].var(),
+                                chains[[col]].var().loc[chain, :])
+                assert_allclose(mcmc.loc[i, col].kurt(),
+                                chains[[col]].kurt().loc[chain, :])
+                assert_allclose(mcmc.loc[i, col].kurtosis(),
+                                chains[[col]].kurtosis().loc[chain, :])
+                assert_allclose(mcmc.loc[i, col].skew(),
+                                chains[[col]].skew().loc[chain, :])
+                assert_allclose(mcmc.loc[i, col].mad(),
+                                chains[[col]].mad().loc[chain, :])
+                assert_allclose(mcmc.loc[i, col].sem(),
+                                chains[[col]].sem().loc[chain, :])
+                assert_allclose(mcmc.loc[i, col].cov(mcmc.loc[i, col]),
+                                chains[[col]].cov().loc[chain, :])
+                assert_allclose(mcmc.loc[i, col].corr(mcmc.loc[i, col]),
```

Comment:
This line is causing a warning

```shell
tests/test_samples.py::test_groupby_stats
  /Users/adam/programming/anesthetic/anesthetic/weighted_pandas.py:266: RuntimeWarning: invalid value encountered in scalar divide
    return self.cov(other, skipna=skipna)/norm
```
Any ideas?

Reply:
Could this come from incorrect/missing dropping of the `'chain'` column?

Reply:
Or possibly from the constant 'minuslogprior` column?

Reply:
Resolved by 706d759